### PR TITLE
Remove "internal error" translations

### PR DESCRIPTION
**Pull request by MichaelChirico**
This also updates messages since the last {R,src} ➡️ .pot extraction; we don't need to update translations yet.

### CODE REVIEW

**Comment by rffontenelle** [comment]
I've spotted at least one untranslated string. Should it be translated in here or after this PR gets merged?

**Comment by MichaelChirico** [comment]
> I've spotted at least one untranslated string. Should it be translated in here or after this PR gets merged?

Let's wait until nearer release. There will be more messages to translate then. The new untranslated strings here are kind of incidental to the removal of no-longer-translated messages.

**Review by rivaquiroga — APPROVED** [review]

**Review by rffontenelle — APPROVED** [review]

**Review by joshhwuu — APPROVED** [review]

**Pull request merged**
